<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuckyRabbit</class>
 <widget class="QMainWindow" name="LuckyRabbit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1874</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LuckyRabbit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Canvas" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>1541</width>
      <height>841</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1780</x>
      <y>940</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Animar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectionButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="moveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="frame">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>940</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>940</y>
      <width>1651</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Marcar quadro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="timeline">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1874</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSalvar_como"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSobre_2">
    <property name="title">
     <string>Sobre</string>
    </property>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuSobre"/>
   <addaction name="menuSobre_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionSalvar_como">
   <property name="text">
    <string>Salvar como</string>
   </property>
  </action>
  <action name="actionEditar">
   <property name="text">
    <string>Editar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
